### PR TITLE
fix(color): align clr/ui with clr/core colors

### PR DESCRIPTION
**Pull request by hippee-lee**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #4546

## What is the new behavior?
Updated colors for @clr/ui

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No
- [x] Maybe - is a very very slight visual change to colors

### CODE REVIEW

File: packages/angular/projects/clr-angular/src/emphasis/alert/_properties.alert.scss [hunk 1]
```diff
@@ -30,8 +30,8 @@
       // Usage: ./_alert.clarity.scss
       --clr-alert-info-bg-color: var(--clr-color-action-50);
       --clr-alert-info-font-color: var(--clr-color-neutral-700);
-      --clr-alert-info-border-color: var(--clr-color-action-400);
-      --clr-alert-info-icon-color: var(--clr-color-action-600);
+      --clr-alert-info-border-color: var(--clr-color-action-800);
```
Comment (mathisscott) [inline packages/angular/projects/clr-angular/src/emphasis/alert/_properties.alert.scss:33]: These changes also need to be made with `./_variables.alert.scss`

File: packages/angular/projects/clr-angular/src/emphasis/_variables.label.scss [hunk 2]
```diff
@@ -10,7 +10,7 @@ $clr-label-borderwidth: $clr-global-borderwidth !default;
 // COLOR
 // Usage:
 // - ./_labels.clarity.scss
-$clr-label-font-color-light: $clr-color-neutral-700 !default;
```
Comment (mathisscott) [inline packages/angular/projects/clr-angular/src/emphasis/_variables.label.scss:13]: These changes also need to be made in `./_properties.label.scss`

File: packages/angular/projects/clr-angular/src/emphasis/_variables.badges.scss [hunk 3]
```diff
@@ -7,13 +7,13 @@
 $clr-badge-font-color-light: $clr-color-neutral-0 !default;
 $clr-badge-font-color-dark: $clr-color-neutral-1000 !default;
 
-$clr-badge-info-bg-color: $clr-color-action-600 !default;
+$clr-badge-info-bg-color: $clr-color-action-800 !default;
```
Comment (mathisscott) [inline packages/angular/projects/clr-angular/src/emphasis/_variables.badges.scss:10]: These changes also need to be made in `./_properties.badges.scss`

File: packages/angular/projects/clr-angular/src/emphasis/_variables.label.scss (hunk 2 above)
Comment (hippee-lee) [reply packages/angular/projects/clr-angular/src/emphasis/_variables.label.scss:13]: Done.

File: packages/angular/projects/clr-angular/src/emphasis/_variables.badges.scss (hunk 3 above)
Comment (hippee-lee) [reply packages/angular/projects/clr-angular/src/emphasis/_variables.badges.scss:10]: Done.

**Pull request merged**